From 3874f5f8d50b2bb36a77bc8beb5263b953e0682a Mon Sep 17 00:00:00 2001
From: Danila Tikhonov <danila@mainlining.org>
Date: Sun, 2 Aug 2026 14:43:44 +0300
Subject: [PATCH 02/40] FROMLIST: arm64: dts: qcom: sm7325-nothing-spacewar:
 Add ath11k calibration-variant

The Nothing Phone (1) has its own calibration files for the WCN6750 used
with the ath11k driver. Add the calibration variant name to the DTS so
that the appropriate calibration files can be loaded from
linux-firmware.

Nothing capitalizes its codenames, so the capitalization is intentional.

Signed-off-by: Danila Tikhonov <danila@mainlining.org>
---
 arch/arm64/boot/dts/qcom/sm7325-nothing-spacewar.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sm7325-nothing-spacewar.dts b/arch/arm64/boot/dts/qcom/sm7325-nothing-spacewar.dts
index e30ee5922..23fd8e1ac 100644
--- a/arch/arm64/boot/dts/qcom/sm7325-nothing-spacewar.dts
+++ b/arch/arm64/boot/dts/qcom/sm7325-nothing-spacewar.dts
@@ -1849,5 +1849,7 @@ &venus {
 };
 
 &wifi {
+	qcom,calibration-variant = "Nothing_Spacewar";
+
 	status = "okay";
 };
